src/views/List.jsx

```diff
@@ -1,16 +1,41 @@
+import { useState } from 'react';
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	const [searchInput, setSearchInput] = useState('');
```

Comment:
Suggestion:  When this state is set, its going to be the item that we want to search in the data for, so perhaps it could be named [searchItem, setSearchItem] for better clarity?

Reply:
Good call, I've changed the variable to `searchInput`.